mslib/mswms/mpl_hsec_styles.py

```diff
@@ -180,6 +180,11 @@ def _plot_style(self):
         mslp = data['air_pressure_at_sea_level']
 
         # Colors in python2.6/site-packages/matplotlib/colors.py
```

Comment:
the comment is likly old

and the prints will make trouble on the server

Reply:
prints were an oversight. removed